docs/v3.x/deployment/google-app-engine.md

```diff
@@ -93,8 +93,6 @@ Create another database, named `strapi` for example. It may be useful to delete
 
 Create the `app.yaml` file in the project root.
 
-Add `app.yaml` to `.gitignore`.
```

Comment:
One might argue that this way database secrets could end up in the repo and be leaked.

Reply:
That's correct, this was removed in error.